gallery/src/components/NxButton/NxButtonIconExample.tsx

```diff
@@ -12,7 +12,7 @@ import { NxButton, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
 const NxButtonIconExample = () =>
   <div className="nx-btn-bar">
     <NxButton>
-      <NxFontAwesomeIcon icon={faSync}/>
+      <NxFontAwesomeIcon aria-hidden={false} aria-label='icon label' icon={faSync}/>
```

Comment:
Should the label be on the icon or the button?  I'll admit it works better than I expected as-is; I was worried it would announce it as "icon label. image. button".  But it skips the "image" part.